src/Components/Component.cpp

```diff
@@ -0,0 +1,109 @@
+// MIT License
```

Comment:
This is not necessary. You can choose to leave it here or remove it, but please be consistent within the project.